tools/experiments/experiments.js

```diff
@@ -163,12 +163,6 @@ const EXPERIMENTS = [
     spec: 'https://github.com/ampproject/amphtml/issues/6106',
     cleanupIssue: 'https://github.com/ampproject/amphtml/pull/6351',
   },
-  {
-    id: 'amp-action-macro',
-    name: 'AMP extension for defining action macros',
-    spec: 'https://github.com/ampproject/amphtml/issues/19494',
-    cleanupIssue: 'https://github.com/ampproject/amphtml/pull/19495',
-  },
```

Comment:
This just removes it from the list on the experiments page. To actually launch the extension, you have to add `"amp-action-macro: 1` to prod-config.json and canary-config.json. I'd revert this change.

Reply:
done. thanks